medusa/server/core.py

```diff
@@ -150,6 +151,8 @@ def __init__(self, options=None, io_loop=None):
             (r'{base}/login(/?)'.format(base=self.options['web_root']), LoginHandler),
             (r'{base}/logout(/?)'.format(base=self.options['web_root']), LogoutHandler),
 
+            (r'{base}/token(/?)'.format(base=self.options['web_root']), TokenHandler),
```

Comment:
This is a fix for now until I get all the auth stuff sorted. Ideally when we login we should be getting a jwt back from Medusa and it should be saved in local storage then vue and the mako ui can access it and use that as a token for the API. Right now we don't do that.